### ISSUE TITLE
Installation of iter8 metrics dashboard fails

### ISSUE THREAD

**Issue by kusumachalasani** [iter8-tools/iter8-istio#378]
**Describe the bug**

Installing metrics dashboard using `curl -L -s https://raw.githubusercontent.com/iter8-tools/iter8-istio/v1.0.1/integrations/grafana/install_dashboard.sh`  fails.

Followed the documentation  as described in  https://github.com/iter8-tools/iter8-istio/blob/master/docs/integrations/grafana.md

The error is because of linking the wrong json location paths.
The install_dashboard.sh script  uses https://raw.githubusercontent.com/iter8-tools/iter8/v1.0.1/integrations/grafana/grafana-v1.json  instead of https://raw.githubusercontent.com/iter8-tools/iter8-istio/v1.0.1/integrations/grafana/grafana-v1.json

**Kubernetes + Istio environment (please provide the following information):**
 - k8s environment: Minikube
 - Istio version: 1.6.8

**Comment by sriumcp** [comment]
Yes, the solution is to alter `iter8` references to `iter8-istio` references in [this script](https://raw.githubusercontent.com/iter8-tools/iter8-istio/v1.0.1/integrations/grafana/install_dashboard.sh). 

@kusumachalasani would you be interested in pushing a PR to address this?

**Comment by kusumachalasani** [comment]
> Yes, the solution is to alter `iter8` GitHub repo references to `iter8-istio` references in [this script](https://raw.githubusercontent.com/iter8-tools/iter8-istio/v1.0.1/integrations/grafana/install_dashboard.sh).
> 
> @kusumachalasani would you be interested in pushing a PR to address this?

sure, will do.

**Comment by kusumachalasani** [comment]
@sriumcp I see these changes are required in the respective version branches and not in the master. 
Change in documentation to use "master" instead of v1.0.1 in the curl commaand `curl -L -s https://raw.githubusercontent.com/iter8-tools/iter8-istio/v1.0.1/integrations/grafana/install_dashboard.sh` will do.
Can I just update the documentation ?

**Comment by sriumcp** [comment]
> Change in documentation to use "master" instead of v1.0.1 in the curl commaand curl -L -s https://raw.githubusercontent.com/iter8-tools/iter8-istio/v1.0.1/integrations/grafana/install_dashboard.sh will do.

Just to confirm, does the `master` branch Grafana installation work as expected for you? In this case, it might make sense to simply create a new patch release `v1.0.2` and then  update the documentation to use `v1.0.2` in place of `v1.0.1` (versioned release artifacts are definitely the more reliable approach rather than installing master).

Please let me know if this solution will work for you.

Yes, update to the documentation after v1.0.2 release would be great.

**Comment by kusumachalasani** [comment]
> Just to confirm, does the `master` branch Grafana installation work as expected for you? 

Yes, it works as expected with the master branch.

> In this case, it might make sense to simply create a new patch release `v1.0.2` and then update the documentation to use `v1.0.2` in place of `v1.0.1` (versioned release artifacts are definitely the more reliable approach rather than installing master).
> Please let me know if this solution will work for you.

Yes, it will.

**Comment by sriumcp** [comment]
@kusumachalasani While working on the above, I realized there were a few other installation fixes that were, most notably those related to dependency pinning. 

Anyway, I pushed a fix that addresses your Grafana issue above as well as the other dependency pinning issues.

The fix is in the master branch. So you original suggestion is best and quickest fix at this point.

> Change in documentation to use "master" instead of v1.0.1 in the curl command curl -L -s https://raw.githubusercontent.com/iter8-tools/iter8-istio/v1.0.1/integrations/grafana/install_dashboard.sh will do.

Please feel free to push the above PR. We will continue to keep this issue open until we are sure the iter8 + Grafana issue is resolved.

@kalantar @Alan-Cha FYI.

**Comment by sriumcp** [comment]
Fixed.
